libraries/user.rb

```diff
@@ -0,0 +1,290 @@
+module MongoDB
```

Comment:
Is this a straight move or are there modifications?

Reply:
Minor mods and I need to test it. However it was too late last night

Reply:
The methods are straight lift and shift.